src/server/pfs/fuse/loopback.go

```diff
@@ -35,12 +35,6 @@ const (
 	dirty                  // we have full content for this file and the user has written to it
 )
 
-func (l *loopbackRoot) setState(mountName, state string) {
-	l.mu.Lock()
-	defer l.mu.Unlock()
-	l.stateMap[mountName] = state
```

Comment:
Is this just dead code? What's the connection between this function and the mount server?

Reply:
That's correct. It's no longer used. It's only caller was in the mount server code.